examples/README.md

```diff
@@ -1,157 +1,55 @@
-it also supports various other endpoints.
-Check the [Controllers](./AspNetCoreMvc/Controllers/) for some
-of the endpoints you can use.
-
-Usage example:
-
-```bash
-aspNetAppTargetFramework="net6.0" ./run-example.sh
-```
-
-### BindingRedirect
-
-Example client app that uses .NET Framework binding redirects to handle assembly
-version conflict between automatic instrumentation and third-party libraries
-that cannot be updated to the same version required by OpenTelemetry.
-See the `OldReference` project for more information.
-
-The project representing the third-party library has a legacy Activity named `InstrumentedHttpCall.GetAsync`
-set [`OTEL_DOTNET_AUTO_LEGACY_SOURCES`](../docs/config.md#customization)
-to capture it.
-
-The following usage example is for Windows, as binding redirection only applies
-to .NET Framework:
-
-```bash
-OTEL_DOTNET_AUTO_LEGACY_SOURCES="InstrumentedHttpCall.GetAsync" exampleApp=BindingRedirect exampleAppTargetFramework=net462 ./run-example.sh
+```sh
+docker-compose build
+docker-compose up -d srv
+docker-compose run cli
 ```
 
-### ConsoleApp
-
-The default example app used by `./run-example.sh`. [OpenTracing](https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/src/OpenTelemetry.Shims.OpenTracing#readme).
-
-Usage example:
+macOS and Linux:
 
-```bash
-exampleAppTargetFramework="net6.0" ./run-example.sh
+```sh
+make
 ```
 
-### ConsoleApp.SelfBootstrap
+The following Web UI endpoints are exposed:
 
-A client app that bootstraps the OpenTelemetry SDK. It uses an
-[ActivitySource](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry/README.md#activity-source)
-to implement manual instrumentation. The bootstrap code takes care of adding
-the ActivitySource name for the automatic instrumentation and the manual instrumentation.
-
-The bootstrap code enables the [Zipkin exporter](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry.Exporter.Zipkin/README.md)
-provided by the SDK. The OpenTelemetry collector container is also configured
-to accept Zipkin spans.
-
-Use the environment variable `exampleAppInjectSDK` in the script to control
-the value of [`OTEL_DOTNET_AUTO_TRACES_ENABLED`](../docs/config.md#customization).
-For example:
-
-```bash
-exampleAppInjectSDK=false exampleApp=ConsoleApp.SelfBootstrap ./run-example.sh
-```
+- <http://localhost:16686/search> - traces
+- <http://localhost:9090/graph> - metrics
 
-### CoreAppOldReference
+You can also find the exported telemetry in the `log` directory.
 
-Example client application that uses NuGet package version resolution at build
-time to handle assembly version conflict between automatic instrumentation
-and third-party libraries that can't be updated to the same version required
-by OpenTelemetry. See the `OldReference` project for more information.
+You can also see the metrics at <http://localhost:8889/metrics>.
```

Comment:
That is true but is a scrape endpoint exposed by the collector, it is not user-friendly. Since Prometheus was already pointed out above what value is added by this info here?

Reply:
OK I will remove it

Side note: I prefer the scrape endpoint exposed by the collector 😆

Reply:
Addressed https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/1708/commits/5c4af5a8c4a2f3e4173b09c3b364f44cec4326f4